build-system/tasks/get-zindex/test.js

```diff
@@ -32,7 +32,8 @@ const result = {
 
 test.cb('collects selectors', t => {
   const data = Object.create(null);
-  m.getZindex('./*.css')
+  const testFiles = `${process.cwd()}/build-system/tasks/get-zindex/*.css`;
```

Comment:
I wonder if you should use `path.resolve()` instead?

Reply:
i switched to __dirname instead